src/Aquifer.Jobs/Subscribers/TranslationMessageSubscriber.cs

```diff
@@ -252,19 +251,38 @@ private async Task TranslateResourceCoreAsync(int resourceContentId,
             throw new InvalidOperationException(
                 $"Aborting translation for Resource Content ID {resourceContentId} because it has no Resource Content Version with Text Media Type in the Draft status.");
```

Comment:
These exceptions should say `aquiferization` instead of `translation` when relevant to avoid confusion.